README.md

```diff
@@ -45,8 +45,14 @@ This states that `process.env.NODE_ENV` must be set to `development`,
 A default value may be provided:
 
 ```javascript
-const NODE_ENV = envvar.enum('NODE_ENV', ['development', 'staging', 'production'], 'production');
+const NODE_ENV = envvar.oneOf('NODE_ENV', ['development', 'staging', 'production'], 'production');
 ```
 
 This states that `process.env.NODE_ENV` must either be unset (in which case the
 default value is assumed), or set to `development`, `staging`, or `production`.
+
+
+### Migration from `1.x` to `2.x`
+
+  * `enum` was renamed to `oneOf` to avoid shadowing the reserved typescript
+    keyword with the same name.
```

Comment:
I don't think it's necessary to mention changes in the readme as users will at least view <https://github.com/plaid/envvar/compare/v1.0.0...v2.0.0> before upgrading the dependency, which will lead them to this pull request.

I'm not opposed to linking to <https://github.com/plaid/envvar/releases> from the readme if you think this would be helpful.